.github/pull_request_template.md

```diff
@@ -1,13 +1,14 @@
 Closes #????
 
-<!--- IF EDITORIAL or CHORE, delete the rest of this template -->
-
 Describe Change Here!
 
-# Issue/PR tracking
-if not applicable state N/A
+<!--- IF EDITORIAL or CHORE, delete the rest of this template -->
 
-<!--- Check these once you have confirmed the related change is not necessary or created an issue/PR and added the link here -->
+# PR tracking
+Check these when the relevant issue or PR has been made, OR after you have confirmed the
+related change is not necessary (add N/A). Leave unchecked if you are unsure. Read the
+[Process Document](https://github.com/w3c/aria/wiki/ARIA-WG-Process-Document/_edit) or
+[Test Overview](https://github.com/w3c/aria/wiki/ARIA-Test-Overview) for more information.
```

Comment:
I don't think the wiki has worked well for the WG so far - most pages are stale, ephemeral content isn't cleaned up well etc. 

I suspect these two pages are important enough to live in the main repository. E.g., a folder of markdown files could provide the same level of ease for users while increasing the chance that the WG will actively maintain it.

If the links stay, the first one should drop the `_edit` suffix.

Reply:
I realize that the use of the wiki is really a separate issue; it can be addressed afterwards if the group wants it.